bitnami/keycloak/templates/configmap-env-vars.yaml

```
@@ -35,7 +35,7 @@ data:
       {{- include "common.tplvalues.render" (dict "value" $path "context" $) }}
     {{- end }}
   {{- end }}
-  {{- if and .Values.ingress.enabled .Values.ingress.hostname }}
+  {{- if and .Values.ingress.enabled .Values.ingress.hostname (empty .Values.proxyHeaders) }}
   KEYCLOAK_HOSTNAME: |-
     {{ ternary "https://" "http://" ( or .Values.ingress.tls (eq .Values.proxy "edge") (not (empty .Values.proxyHeaders)) ) -}}
```

Here `proxyHeaders` will be always empty, so we can remove it:
```suggestion
    {{ ternary "https://" "http://" ( or .Values.ingress.tls (eq .Values.proxy "edge") ) -}}
```

---

Yes, you are right. I removed it.